src/items/implementations.md

```diff
@@ -51,7 +51,7 @@ The nominal type is called the _implementing type_ and the associable items are
 the _associated items_ to the implementing type.
 
 Inherent implementations associate the contained items to the implementing type.
-The associated item has a path of a path to the implementing type followed by
+The path of an associated item consists of a path to the implementing type followed by
```

Comment:
Hmm... I don't find the current nor the changed wording to be great. In particular "path" gets repeated twice in succession; can we find an alternate wording?

Reply:
I don't think we can really get away from repeating path here, since it is the proper descriptive word.  This sentence is defining how to construct the path to an item associated with a trait, and part of that path is the path to the implementing type.  IMO using a synonym for path just to avoid the repetition would be even less clear.

That being said, if you have a concrete suggestion on how to improve it I'm all for it :)

Reply:
I got nothing... sorry :)